docs/getting-started/run.mdx

```diff
@@ -53,9 +57,9 @@ Check if everything works by running `flutter run` with the newly created Widget
 />
 
 <Success>
-  Widgetbook is now running, but no components are shown yet. To learn how to add components and how
-  to configure Widgetbook to your liking, view the [configuration
-  documentation](/getting-started/configuration).
+  Widgetbook is now running, but no components are shown yet. To learn how to
+  add components and how to configure Widgetbook to your liking, view the
+  [configuration documentation](/getting-started/configuration).
 </Success>
 
 You can check the final getting started on Zapp!
```

Comment:
Feature Request for Zapp: It would be cool if it looked like the other code snippets, but had a little button in the bottom or top right corner: "Run in Zapp".